pkg/kubelet/server/stats/summary.go

```diff
@@ -198,8 +209,12 @@ func (sb *summaryBuilder) buildSummaryPods() []stats.PodStats {
 			podToStats[ref] = podStats
 		}
 
+		glog.V(4).Infof("STATS buildSummaryPods - podStats %v", podStats)
+
 		// Update the PodStats entry with the stats from the container by adding it to stats.Containers
 		containerName := dockertools.GetContainerName(cinfo.Spec.Labels)
+		glog.V(4).Infof("STATS buildSummaryPods - containerName %v", containerName)
```

Comment:
I'm a little worried about these log messages becoming to spammy, even on V(4). Could you at least reduce it to a single log entry with all the required info?